24life/blocks/newsletter-form/newsletter-form.js

```diff
@@ -0,0 +1,45 @@
+// eslint-disable-next-line no-unused-vars
+export default function decorate(block) {
+  block.innerHTML = `<form id="subForm" style="line-height: 1.5em;" action="http://newsletter.culturefoundry.com/t/r/s/hruuutt/" method="post">
```

Comment:
```suggestion
  block.innerHTML = `<form id="subForm" style="line-height: 1.5em;" action="https://newsletter.culturefoundry.com/t/r/s/hruuutt/" method="post">
```

Reply:
Otherwise my chrome refuses to send POST data.

Reply:
fixed